pkgs/development/tools/revive/default.nix

```diff
@@ -33,16 +33,15 @@ buildGoModule rec {
     ldflags+=" -X 'github.com/mgechev/revive/cli.date=$(cat DATE)'"
   '';
 
-  # The following tests fail when built by nix:
-  #
-  # $ nix log /nix/store/build-revive.1.3.9.drv | grep FAIL
-  #
-  # --- FAIL: TestAll (0.01s)
-  # --- FAIL: TestTimeEqual (0.00s)
-  # --- FAIL: TestTimeNaming (0.00s)
-  # --- FAIL: TestUnhandledError (0.00s)
-  # --- FAIL: TestUnhandledErrorWithBlacklist (0.00s)
-  doCheck = false;
+  allowGoReference = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ go ];
```

Comment:
```suggestion
```

Reply:
I guess the `wrapProgram` call handles it?

Reply:
Since you are already explicitly referencing go in `postFixup`, there's no need for it to be in `buildInputs`

Reply:
Thanks. I've removed `buildInputs` and rebased and force pushed.